postman-generator/src/test/scala/generator/PostmanCollectionGeneratorSpec.scala

```diff
@@ -77,7 +77,7 @@ class PostmanCollectionGeneratorSpec extends WordSpec with Matchers {
           |  },
           |  "variable" : [ {
           |    "type" : "string",
-          |    "value" : "https://api.flow.io",
+          |    "value" : "",
```

Comment:
it would be good to add a `baseUrl` to our `trivialService` that is used in this test case.
So the value is not empty.

You can also add a short test case for the empty case if you think it'd be beneficial.
test on a parsed model, not the string assertions
(e.g. `collection.variable shouldEqual Seq(Variable(_, "")))`